### DISCUSSION TITLE
Disable LazyVim LSP keymap

### DISCUSSION THREAD

**Discussion by Silzinc** [LazyVim/LazyVim#4834] [Q&A] [answered]
Hello ! I would like to disable the signature help on the <c-k> key in insert mode. I tried adding the following to keymaps.lua without success. I do not touch the key anywhere else in my config. Thanks in advance for your help!

```lua
local keys = require("lazyvim.plugins.lsp.keymaps").get()
keys[#keys + 1] = { "<C-k>", false, mode = { "i" } }
```

**Comment by dpetka2001** [comment]
The docs on the website have an example about this. You should read them. On the phone now so can't write code snippet. https://www.lazyvim.org/plugins/lsp#%EF%B8%8F-customizing-lsp-keymaps

**Reply by Silzinc** [reply]
Thank you for the fast answer. However, I did try that in keymaps.lua already without success. Doing it in the nvim-lspconfig lazy options did not change anything either.

**Reply by dpetka2001** [reply]
You don't do that in ` keymaps.lua`. You create a file in `/lua/plugins/` directory and you put the example there just like all plugin specs. Kindly read the docs to understand how things work.

**Reply by Silzinc** [reply]
Yes, I did that too, that's what I meant by "Doing it in the nvim-lspconfig lazy options". I tried making opts a function and put that in it, as well as putting it in the init function of the config.

**Reply by dpetka2001** [reply]
Can you show the whole code and where you created the file?

**Reply by Silzinc** [reply]
plugins/coding/lspconfig.lua
```lua
return {
	{
		"neovim/nvim-lspconfig",
		opts = function(_, opts)
			opts.inlay_hints = { enabled = false }
			local keys = require("lazyvim.plugins.lsp.keymaps").get()
			keys[#keys + 1] = { "<C-k>", false, mode = { "i" } }
			return opts
		end,
	},
}
```
In config/lazy.lua, I made sure to import the coding directory (I have other config filed in that subdir that work perfectly fine).

**Reply by dpetka2001** [reply]
Does it work if you put it in `/lua/plugins/lspconfig.lua`?

**Reply by Silzinc** [reply, marked as answer]
Interestingly, after deleting and redownloading my config, the issue was solved with the file in the coding directory. Well I do not know what happened, thanks for the help still!
